javascript/js-forms_calculator/js/index.js

```diff
@@ -26,6 +26,26 @@ form.addEventListener("submit", (event) => {
 
   // --v-- write your code here --v--
 
+  const numberA = parseFloat(form.elements.numberA.value);
+  const numberB = parseFloat(form.elements.numberB.value);
+
+  const operator = form.elements.operator.value;
+
+  if (operator === "addition") {
```

Comment:
Is not wrong per se. But use `else if` statement instead of other `if`.
In this case you could have use `switch`